index.html

```diff
@@ -290,7 +290,23 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-
+      <footer class="footer">
+        <div class="footer-div">
+          <div class="fheading">
+            <h2 class="contact-me text-white text-center">Contact me</h2>
+            <p class="para text-white">If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with it.</p>
+          </div>
+          <form action="https://formspree.io/f/xrgranwn" class="form" method="POST">
+            <input name="username" maxlength="30" class="mb-4 uname" id="name" type="text" placeholder="your name" required>
+            <input name="email" id="email" class="mb-4 email" type="email" placeholder="example@gmail.com" required>
```

Comment:
Great job on adding the validation on the other <input> and <textarea> tags. You may have forgotten to include it for this tag.